docs/content/en/third-party/_index.md

```diff
@@ -10,19 +10,17 @@ K8ssandra provides a production-ready platform for running Apache Cassandra&reg;
-|reaper-operator                                                      |19d15e499171fe712aef3657be3d59b3044b57fa                                  |Apache-2.0                                                                                                                                                                                                                                                                                                                                                                    |
-|terratest                                                            |b13ada22754d4ec78aa0f3a838f9030071fee22b                                  |Apache-2.0; public-domain	                                                                                                                                                                                                                                                                                                                                                       |
-|yaml                                                                |9fc95527decd95bb9d28cc2eab08179b2d0f6971                                   |Multi-license: BSD-3-Clause OR MIT; BSD-3-Clause                                                                                                                                                                                                                                                                                                                                 |
+For each component deployed by K8ssandra, see the following 3rd-party license reports. Each one was generated by the Fossa app. Start in the #dependencies section of each component's GitHub-hosted README. From there, access the latest generated "open source report" link.
+
+* [K8ssandra dependencies]({{< relref "https://github.com/k8ssandra/k8ssandra#dependencies" >}}) 
+* [Cass Operator dependencies]({{< relref "https://github.com/k8ssandra/cass-operator/#dependencies" >}})
+* [Medusa Operator dependencies]({{< relref "https://github.com/k8ssandra/medusa-operator/#dependencies" >}})
+* [Reaper Operator dependencies]({{< relref "https://github.com/k8ssandra/reaper-operator/#dependencies" >}})
+* [Management API for Apache Cassandra dependencies]({{< relref "https://github.com/k8ssandra/management-api-for-apache-cassandra/#dependencies" >}})
+* [Go client library for the Reaper for Apache Cassandra REST API dependencies]({{< relref "https://github.com/k8ssandra/reaper-client-go/#dependencies" >}})
+* [Cassandra Reaper dependencies]({{< relref "https://github.com/thelastpickle/cassandra-reaper/#dependencies" >}})
+* [Cassandra Medusa dependencies]({{< relref "https://github.com/thelastpickle/cassandra-medusa/#dependencies" >}})
```

Comment:
I would remove "dependencies" from each of the links.

Reply:
```suggestion
* [K8ssandra]({{< relref "https://github.com/k8ssandra/k8ssandra#dependencies" >}}) 
* [Cass Operator]({{< relref "https://github.com/k8ssandra/cass-operator/#dependencies" >}})
* [Medusa Operator]({{< relref "https://github.com/k8ssandra/medusa-operator/#dependencies" >}})
* [Reaper Operator]({{< relref "https://github.com/k8ssandra/reaper-operator/#dependencies" >}})
* [Management API for Apache Cassandra]({{< relref "https://github.com/k8ssandra/management-api-for-apache-cassandra/#dependencies" >}})
* [Go client library for the Reaper for Apache Cassandra REST API]({{< relref "https://github.com/k8ssandra/reaper-client-go/#dependencies" >}})
* [Cassandra Reaper]({{< relref "https://github.com/thelastpickle/cassandra-reaper/#dependencies" >}})
* [Cassandra Medusa]({{< relref "https://github.com/thelastpickle/cassandra-medusa/#dependencies" >}})
```

Reply:
Will do.